### PR TITLE
change exported COLOR to ICON_COLOR in Icons

### PR DESCRIPTION
**Pull request by matikrk**
part of issue #1334
* all Icon-* files import form Icon.jsx
* create mixin in scss for coloring icons
* all exported thing with name ICON_COLOR point to same object

### CODE REVIEW

**Comment by matikrk** [comment]
working :)

File: src/components/icon-as-button/_icon-as-button.scss [hunk 1]
```diff
@@ -166,6 +170,33 @@ $includeHtml: false !default;
...
+
+      &:hover,
+      &:active {
+        fill: $iconAsButtonBlueActiveColor;
+      }
+
+      .sg-icon-as-button__hole {
+        border-color: $iconAsButtonBlueColor;
+      }
+    }
```
Comment (kdzwinel) [inline src/components/icon-as-button/_icon-as-button.scss]: this should be in a separate PR, right?

Comment (matikrk) [reply src/components/icon-as-button/_icon-as-button.scss]: i add mising color to icaon_as _button to have same everywhere

**Comment by kdzwinel** [comment]
Have you checked if interactive version of docs works correctly? Have you added new colors to docs?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "style-guide",
-  "version": "121.0.1",
+  "version": "122.0.0",
```
Comment (kdzwinel) [inline package.json]: this is a breaking change - you can't import "COLOR" from "icons/Icon"

**Comment by matikrk** [comment]
interactive and normal working. Colors is based on array variable

File: package.json (hunk 2 above)
Comment (matikrk) [reply package.json]: breaking change = major - so what it's wrong?
